components/automate-ui/src/app/pages/+compliance/+reporting/+reporting-node/reporting-node.component.ts

```diff
@@ -235,12 +236,14 @@ export class ReportingNodeComponent implements OnInit, OnDestroy {
     }
   }
 
-  openControlPane(control: { id: string | number; }) {
-    return this.openControls[control.id].pane;
+  openControlPane(control: { id: string | number, profile_id: string; }) {
+    const key = control.id + ' ' + control.profile_id;
```

Comment:
can we have a function that takes id and profile_id as input and return the key? so that we can maintain the combining logic in one place. And we can use that function where ever we are preparing the key. Line 190,197, 240, 245, and also at line number 227 in components/automate-ui/src/app/pages/+compliance/+reporting/+reporting-node/reporting-node.component.html

Reply:
done